### PR TITLE
Feature/e2e 1662

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,9 @@
   "private": true,
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
+    "@apollo/react-testing": "^3.1.3",
     "@testing-library/jest-dom": "^5.1.1",
-    "@testing-library/react": "^9.5.0",
+    "@testing-library/react": "^10.0.2",
```
Comment (ghost) [inline package.json:9]: new testing library had breaking changes but it is faster

File: package.json [hunk 2]
```diff
@@ -63,7 +65,7 @@
   "scripts": {
     "start": "SKIP_PREFLIGHT_CHECK=true craco start",
     "build": "SKIP_PREFLIGHT_CHECK=true craco build",
-    "test": "SKIP_PREFLIGHT_CHECK=true craco test --watchAll",
+    "test": "SKIP_PREFLIGHT_CHECK=true craco test --env=jest-environment-jsdom-sixteen --watchAll",
```
Comment (ghost) [inline package.json:68]: this is a workaround for new version of testing library, will replace when they have a solution

**Comment by ghost** [comment]
current coverage, will keep updating as it improves
Screenshot 2020-03-31 at 6 22 51 PM

File: package.json (hunk 2 above)
Comment (rachit-aasaanjobs) [reply package.json:68]: please add a todo comment here.

File: craco.config.js [hunk 3]
```diff
@@ -34,4 +34,14 @@ module.exports = {
     eslint: {
         mode: ESLINT_MODES.file
     },
+    jest: {
```
Comment (rachit-aasaanjobs) [inline craco.config.js:37]: it would be better if you make a different file for config. check croco configurations

File: package.json (hunk 2 above)
Comment (ghost) [reply package.json:68]: I believe unfortunately we can't add  comments in JSON, where else do you think adding a comment for this would be meaningful?

**Review by rachit-aasaanjobs — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (rachit-aasaanjobs) [reply package.json:68]: its ok, just keep in mind we have to update it

**Pull request merged**